examples/to_mdbx.rs

```diff
@@ -0,0 +1,150 @@
+//! Convert a block folder to a MDBX folder
```

Comment:
This seems to be too important to be put under `examples`. Until we have a better structure it's easier to just put it in `tests`, to be imported in automated tests & benchmarks (not manually).